make_test_images/Cargo.toml

```diff
@@ -13,14 +13,14 @@ c2pa = { path = "../sdk", default-features = false, features = [
 	"unstable_api",
 ] }
 env_logger = "0.11"
-log = "0.4.8" 
-image = { version = "0.25.1", default-features = false, features = [
```

Comment:
I've pinned this to `0.25.1` to fix the tests - since Clippy would fail with the current version since we are now using a deprecated API (there is now an `ImageReader` that takes the place of `io::Reader`) and the minimum version tests would fail if I updated the API.